packages/merge-on-green/src/merge-logic.ts

```diff
@@ -495,6 +495,26 @@ mergeOnGreen.commentOnPR = async function commentOnPR(
   }
 };
 
+mergeOnGreen.checkPRMerged = async function checkPRMerged(
+  owner: string,
+  repo: string,
+  pr: number,
+  github: GitHubAPI
+): Promise<boolean> {
+  try {
+    await github.pulls.checkIfMerged({
```

Comment:
Does this function raise an error if the PR is not merged? Seems like it should return a boolean-type value? Docs aren't great for this: https://octokit.github.io/rest.js/v17#pulls-check-if-merged.

Reply:
Yeah, it's a bit strange.... Here's the best version of the documentation I could find:
https://developer.github.com/v3/pulls/#get-if-a-pull-request-has-been-merged

I've tested it myself as well and that seems to be the functionality. Ben recommended we print out the error just so we're aware and not just assuming that it means the PR has not been merged. I agree though, it's a strange method :)

Reply:
That's surprising! But, sounds good!